napari/utils/theme.py

```diff
@@ -294,7 +294,7 @@ def rebuild_theme_settings():
                 'warning': 'rgb(153, 18, 31)',
                 'current': 'rgb(0, 122, 204)',
                 'syntax_style': 'native',
-                'console': 'rgb(0, 0, 0)',
+                'console': 'rgb(18, 18, 18)',
```

Comment:
follow material design suggestions to not use the black background. For me looks better 
https://material.io/design/color/dark-theme.html#properties